docs/tutorials/Calibrating a single-qubit gates on a pulse backend_.rst

```diff
@@ -0,0 +1,352 @@
+whether a values is valid or not and the experiment id that produce a value.
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[qubit]))[columns_to_show]
+    
+    
+=================================================================
+2. Calibrating the pulse amplitudes with a Rabi experiment
+=================================================================
+In the Rabi experiment we apply a pulse at the frequency of the qubit 
+and scan its amplitude to find the amplitude that creates a rotation 
+f a desired angle. We do this with the calibration experiment ``RoughXSXAmplitudeCal``.
+This is a specialization of the ``Rabi`` experiment that will update the calibrations 
+for both the ``X`` pulse and the ``SX`` pulse using a single experiment.
+
+.. jupyter-execute:: 
+
+    from qiskit_experiments.library.calibration import RoughXSXAmplitudeCal
+    rabi = RoughXSXAmplitudeCal(qubit, cals, backend=backend, amplitudes=np.linspace(-0.1, 0.1, 51))
+
+The rough amplitude calibration is therefore a Rabi experiment in which 
+each circuit contains a pulse with a gate. Different circuits correspond to pulses 
+with different amplitudes.
+
+.. jupyter-execute::
+
+    rabi.circuits()[0].draw("mpl")
+
+After the experiment completes the value of the amplitudes in the calibrations 
+will automatically be updated. This behaviour can be controlled using the ``auto_update``
+argument given to the calibration experiment at initialization.
+
+.. jupyter-execute::
+
+    rabi_data = rabi.run().block_for_results()
+    rabi_data.figure(0)
+
+.. jupyter-execute::
+
+    print(rabi_data.analysis_results("rabi_rate"))
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[qubit, ()], parameters="amp"))[columns_to_show]
+
+The table above shows that we have now updated the amplitude of our :math:`pi` pulse 
+from 0.5 to the value obtained in the most recent Rabi experiment. 
+Importantly, since we linked the amplitudes of the ``x`` and ``y`` schedules 
+we will see that the amplitude of the ``y`` schedule has also been updated 
+as seen when requesting schedules form the ``Calibrations`` instance. 
+Furthermore, we used the result from the Rabi experiment to also update 
+the value of the ``sx`` pulse. 
+
+.. jupyter-execute::
+
+    cals.get_schedule("sx", qubit)
+
+.. jupyter-execute::
+
+    cals.get_schedule("x", qubit)
+   
+.. jupyter-execute::
+
+    cals.get_schedule("y", qubit)
+
+
+=====================================
+3. Saving and loading calibrations
+=====================================
+The values of the calibrated parameters can be saved to a .csv file 
+and reloaded at a later point in time. 
+
+.. jupyter-execute::
+
+    cals.save(file_type="csv", overwrite=True, file_prefix="PulseBackend")
+
+After saving the values of the parameters 
+you may restart your kernel. If you do so, you will only need to run the 
+following cell to recover the state of your calibrations.
```

Comment:
```suggestion
After saving the values of the parameters you may restart your kernel. If you do so, 
you will only need to run the following cell to recover the state of your calibrations. 
Since the schedules are currently not stored we need to call our ``setup_cals`` function 
or use a library to populate an instance of Calibrations with the template schedules. 
By contrast, the value of the parameters will be recovered from the file.
```